### PR TITLE
[Docs] Describe more about --dir option for wasmedge

### PR DESCRIPTION
**Pull request by harry900831**

### CODE REVIEW

File: docs/book/en/src/cli/wasmedge.md [hunk 1]
```diff
@@ -213,7 +214,7 @@ The [hello.js](https://github.com/WasmEdge/WasmEdge/raw/master/examples/js/hello
 You can run:
 
 ```bash
-wasmedge --dir .:. qjs.wasm hello.js 1 2 3
+wasmedge --dir . qjs.wasm hello.js 1 2 3
```
Comment (hydai) [inline docs/book/en/src/cli/wasmedge.md:217]: We should keep both examples and tell users that they are the same.

File: docs/book/en/src/cli/wasmedge.md [hunk 2]
```diff
@@ -41,7 +41,8 @@ The options of the `wasmedge` CLI tool are as follows.
     * WasmEdge will execute the function which name should be given in `ARG[0]`.
     * If there's exported function which names `_initialize`, the function will be executed with the empty parameter at first.
 4. (Optional) `--dir`: Bind directories into WASI virtual filesystem.
-    * Use `--dir guest_path:host_path` to bind the host path into the guest path in WASI virtual system.
+    * Use `--dir host_path` to bind the host path in WASI virtual system.
```
Comment (hydai) [inline docs/book/en/src/cli/wasmedge.md:44]: We should show the equal mapping with `:` format.

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/en/src/cli/wasmedge.md (hunk 1 above)
Comment (harry900831) [reply docs/book/en/src/cli/wasmedge.md:217]: Done

File: docs/book/en/src/cli/wasmedge.md (hunk 2 above)
Comment (harry900831) [reply docs/book/en/src/cli/wasmedge.md]: Done

**Review by hydai — APPROVED** [review]

**Pull request merged**